src/components/ec/base/ucc_ec_base.h

```diff
@@ -67,6 +67,7 @@ typedef enum ucc_ee_executor_task_type {
     UCC_EE_EXECUTOR_TASK_TYPE_REDUCE             = UCC_BIT(1),
     UCC_EE_EXECUTOR_TASK_TYPE_REDUCE_MULTI       = UCC_BIT(2),
     UCC_EE_EXECUTOR_TASK_TYPE_REDUCE_MULTI_ALPHA = UCC_BIT(3),
+    UCC_EE_EXECUTOR_TASK_TYPE_COPY_MULTI         = UCC_BIT(4),
```

Comment:
do we need both   UCC_EE_EXECUTOR_TASK_TYPE_COPY and  UCC_EE_EXECUTOR_TASK_TYPE_COPY_MULTI? can COPY be simple case of multi with num_verctors=1?

Reply:
i think we want to keep it separate because it will allow optimizing the signalling when executar task goes to CUDA EC. TASK_COPY is smaller, less data to fetch over PCI (when executor kernel reads the task from host). @Sergei-Lebedev  other reasons?

Reply:
ic. i thought unifying might reduce the ucc_ee_executor_task_args_t size  by removing uf[]

Reply:
there is pending PR (#542) that refactors all this and different task types will go under union. That PR will be rebased when this one is merged